docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -1,26 +1,40 @@
+
+Nodes of a single service (collectively called a cluster) require less decoupling. They share the same code and are managed together, as a set, by a single team or individual. For this reason, intra-service communication can take advantage of mechanisms that have less overhead and better performance. For example:
+
+* Many Lagom components use Akka remoting internally, and you can use it directly in your services.
+
+* [[Distributed publish-subscribe|PubSub]] can be used for low-latency, at-most-once messaging between nodes. Limitations include:
 <ul><ul>
 <li>Network interruptions can cause messages to be lost. </li>
 <li>When a new instance starts, joins the cluster, and subscribes, it will not receive messages sent before its subscription.</li>
 </ul></ul>  
 
-* Publish with the [[Message Broker API|MessageBrokerAPI]] --- for at-least-once semantics. If a new instance starts publishing information, its messages are added to events previously emitted. If a new instance subscribes to a topic, they will receive all events, past, present, and future (as long as they are subscribed). Lagom implements the API to work with [[Kafka|KafkaServer]], but you could also implement your own. 
+* Databases and other persistent storage can be seen as another way that nodes of a service communicate.For microservices that use persistent entities, Lagom encourages [[event streaming|ES_CQRS]], which also takes advantage of asynchronous communication and provides guarantees via an event log.
+
+This diagram illustrates each of these types of inter- and intra-service communication in a Lagom system distributed across three servers. In the example, the Order service publishes to one or more Kafka topics, while the User service subscribes to consume the information. The User service communicates with other User service instances (cluster members) using Akka remoting. The Shipping service and User service exchange information by streaming it in service calls. [[ServiceCommunication.png]] 
+
+## Communication with parties outside of a microservices system
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For such extra-service communication, third parties can obtain data asychronously from Lagom services that publish to the Broker API and enjoy the at-least-once guarantee. Lagom services also expose an API for third parties to exchange data synchronously. This is usually mapped to HTTP. Lagom Service APIs also support streaming data to external clients by means of websockets. See [[ServiceDescriptors]] for more information. 
```

Comment:
This paragraph doesn't totally make sense to me. "For such extra-service communication" seems to have a missing referent. Is this talking about communication with internet clients, like the next paragraph, or something else? Communication with internet clients would not use the Message Broker API, it would always use service calls.

If it means communication with legacy systems, that makes more sense, but I personally wouldn't think of that as "external" communication. I would consider the legacy system to be part of the system of microservices. Maybe that's just me 😄